Doxyfile

```diff
@@ -2165,7 +2165,7 @@ MAN_LINKS              = NO
 # captures the structure of the code including all documentation.
 # The default value is: NO.
 
-GENERATE_XML           = YES
+GENERATE_XML           = NO
```

Comment:
Why did this need to be changed?

Reply:
Doxygen will output the docs in both xml format and html format.
-> We are not using the xml output, and it makes my git explode. >:c